client/src/views/Models/Model.vue

```diff
@@ -396,20 +195,59 @@
     drilldownRelatedParameters: any = null;
     drilldownParameters: any = null;
 
-    isSplitView = false;
     subgraph: GraphInterface = null;
+    querygraph: any = null;
     showModalParameters: boolean = false;
     showModalMetadata: boolean = false;
     modalDataParameters: any = null;
     modalDataMetadata: any = null;
     highlights: SubgraphInterface = null;
-    pathsCounter: number = 0;
 
     @Getter getSelectedModelIds;
     @Getter getModelsList;
     @Getter getParameters;
+    @Getter getFilters;
     @Mutation setSelectedModels;
 
+    @Watch('getFilters') onGetFiltersChanged (): void {
+      this.executeFilters();
+    }
+
+    mounted (): void {
+      this.loadData();
+    }
+
+    executeFilters (): void {
+      if (this.bgraphInstance && this.getFilters?.clauses.length > 0) {
+        const querygraph = filterToBgraph(this.bgraphInstance, this.getFilters);
+        this.querygraph = {
```

Comment:
`queryGraph` instead to follow camelCase notation.

Reply:
Let's address this on the merge @mj3cheun. I have a feeling this name won't stick around.

Reply:
Switched this to `subgraph`